frontend/app/views/fragments/oauth/staffUnauthorisedError.scala.html

```diff
@@ -1,6 +1,13 @@
 @import configuration.Config
 
-<p>
-    If you are having problems signing up for your free Partner membership please email
-    <a href="mailto:@Config.membershipSupportStaffEmail">@Config.membershipSupportStaffEmail</a>.
-</p>
+<p>Unfortunately your email address has not been added to the approved staff list. This is probably for one of the following reasons:</p>
+
+<h2>1. You are not a permanent employee or fixed term contractor</h2>
+
+<h2>2. You have joined the Guardian in the last month</h2>
+
+<p>We use a Google group to authenticate email addresses. The group is managed by HR and updated once a month.</p>
+<p>If you've recently joined The Guardian as permanent employee or fixed term contractor then please try and register again at the beginning of next month.</p>
+<p>If you have an urgent requirement please email <a href="mailto:hr.admin@@theguardian.com">hr.admin@@theguardian.com</a> and ask to be added to the staff google group - Once this is done you will be able to join.</p>
```

Comment:
Going to get this out I think. Happy for it to not be in config as we're only using it once.